tests/test_ci/TestControllers.py

```diff
@@ -709,8 +710,235 @@ def test_webhook_pr_opened(self, mock_request, mock_queue_test, mock_github, moc
 
         self.assertEqual(response.data, b'{"msg": "EOL"}')
         mock_blocked.query.filter.assert_called_once_with(mock_blocked.user_id == 'test')
+        mock_add_test_entry.assert_called_once()
+
+    @mock.patch('mod_ci.controllers.GitHub')
+    @mock.patch('mod_ci.controllers.schedule_test')
+    @mock.patch('requests.get', side_effect=mock_api_request_github)
+    def test_webhook_workflow_run_event_requested_action_with_valid_workflow_name(self, mock_request,
```

Comment:
```suggestion
    def test_webhook_valid_workflow_name(self, mock_request,
```
let's try to shorten the test names?

Reply:
> let's try to shorten the test names?

I have shortened the names, further shortening might lead to confusion between tests, as there are many possible cases within workflow run event.😅